internal/venice-client-common/src/main/java/com/linkedin/venice/utils/metrics/MetricsRepositoryUtils.java

```diff
@@ -45,9 +61,17 @@ public static MetricsRepository createSingleThreadedMetricsRepository(
 
   public static MetricsRepository createSingleThreadedVeniceMetricsRepository(
       long maxMetricsMeasurementTimeoutMs,
-      long initialMetricsMeasurementTimeoutMs) {
+      long initialMetricsMeasurementTimeoutMs,
+      boolean isOtelEnabled,
+      VeniceOpenTelemetryMetricNamingFormat otelFormat) {
+    Collection<MetricEntity> metricEntities = new ArrayList<>();
+    metricEntities
+        .add(new MetricEntity("test_metric", MetricType.HISTOGRAM, MetricUnit.MILLISECOND, "Test description"));
```

Comment:
That does not seem right within the `main` src. Is it leftover of some debugging? Should it be cleaned up? If there is a reason to keep it, then please add a comment explaining why.

Reply:
Good catch, moved this code away from here.